app/code/Magento/InventoryGroupedProduct/etc/module.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
+    <module name="Magento_InventoryGroupedProduct" setup_version="1.0.0" >
+        <sequence>
+            <module name="Magento_InventoryConfiguration" />
```

Comment:
We don't need sequence on `Magento_InventoryConfiguration`

Reply:
Removed.